specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/DeletedServers.json

```diff
@@ -142,61 +134,10 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/deletedServers/{deletedServerName}/recover": {
```

Comment:
do we plan to delete this API path? It will be breaking change

Reply:
Yes. This API path is already deprecated. We keep trying to remove it from the docs (and it keeps coming back :/ )

Reply:
"Each new preview version of the service must be backward compatible with the latest GA version. However, a new 
preview or GA version can make breaking changes as compared to the previous preview version. Once a new 
preview or GA version is available, notify customers that the previous preview version will stop working entirely in 
90 days."

I don't understand this. How can a new preview make breaking changes as compared to the previous preview version, but also NOT make breaking changes with the latest GA version?

Reply:
Ah. I see that we tried to remove this API a year ago (it currently does nothing), but never made it to this repository.